static/openapi/prerendering-spec.json

```diff
@@ -4335,525 +4691,631 @@
+                      },
+                      {
+                        "timestamp": 1672542000,
+                        "origin": "https://www.origin.com",
+                        "path": "/terms",
+                        "bytesSent": 362056,
+                        "responseTime": 1500,
+                        "isMobile": false,
+                        "responseCode": 200
+                      }
+                    ]
+                  }
                 }
               }
             }
           },
           "400": {
-            "description": "Bad request",
+            "description": "Bad request.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Bad request",
+                  "description": "Bad request.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
+                  }
                 }
               }
             }
           }
         }
       }
     },
-    "/api/prerender/v1/render/{mobile}/{url}": {
+    "/api/prerender/v1/history/render/{origin}": {
```

Comment:
should be under metric

Reply:
The history API is outside metrics.
This one is to bring the list of renders over time.